kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/DomainMetadataSuite.scala

```diff
@@ -227,30 +233,6 @@ class DomainMetadataSuite extends DeltaTableWriteSuiteBase with ParquetSuiteBase
     }
   }
 
-  test("multiple DomainMetadatas for the same domain should fail in single transaction") {
```

Comment:
we remove this test because we now fail earlier in all APIs right? just clarifying

Reply:
Previously there was the `withDomainMetadata` in the TransactionBuilder that would overwrite if there were duplicate domain metadatas.

Then there was the `VisibleForTesting` one that allowed tests and internal classes to put new domain metadatas into the domain metadatas list. This test was checking that with the `visibleForTesting`, it would fail in the `commit` if there were duplicates.

 This is no longer the semantics. It was also testing an internal interface. This is why I thought I could remove this test.